pkg/kv/kvserver/client_replica_circuit_breaker_test.go

```diff
@@ -119,6 +120,15 @@ func TestReplicaCircuitBreaker(t *testing.T) {
 		tc.RequireIsNotLeaseholderError(t, tc.Read(n2))
 		tc.RequireIsNotLeaseholderError(t, tc.Write(n2))
 	})
+
+	runCircuitBreakerTest(t, "leaseholder-unavailable", func(t *testing.T, ctx context.Context, tc *circuitBreakerTest) {
```

Comment:
This should probably go into the next commit, "wip: add leaseholder-unavailable test".